contracts/PetPark.sol

```diff
@@ -1,7 +1,201 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
+contract PetPark
+{
+    // -- SCAFFOLDING -------------------------
+    enum AnimalType
+    {
+        NONE
+    ,   FISH
+    ,   CAT
+    ,   DOG
+    ,   RABBIT
+    ,   PARROT
+    }
 
-contract PetPark {
+    struct Borrower
+    {
+        uint8      age;
+        AnimalType animal;
+        bool       isFemale;
```

Comment:
This can be gender and you can use enum to store it similar to `AnimalType`

Reply:
yes; a bool is faster, no?

Reply:
We don't optimise for time in solidity (mentioned the reason in my other comment). The optimisation should be for gas and storage